### ISSUE TITLE
GPT-4 MMLU evaluation setup

### ISSUE THREAD

**Issue by brucewlee** [allenai/open-instruct#23]
Dear authors,

I enjoyed reading your work! I wonder if the GPT-4 evaluation results described in the paper only evaluate 100 instances from each MMLU subject? I ask this because it seems like only 100 instances are used per subject, on your mmlu.sh evaluation script.

If this is true, why wasn't this discussed in the paper? That is, is it a convention to use only 100 instances per subject for GPT-4 evaluation?

Thanks

**Comment by yizhongw** [comment]
Hi @brucewlee, thanks for raising this question! Your observation is true - for evaluating GPT4's performance on MMLU, we only used 100 instances per subject (totally 57 * 100 instances). We did this only for GPT4 model, mainly to save the time/cost of this evaluation. From my experience, the numbers should stay quite similar. I don't think this is a common practice. We will make sure to clarify this in our next version.
